alonzo/impl/src/Cardano/Ledger/Alonzo/Translation.hs

```diff
@@ -205,7 +206,7 @@ translateTxOut (Shelley.TxOutCompact addr value) =
 -- (PParamsUpdate era) = (PParams' StrictMaybe era)
 
 translatePParams ::
-  AlonzoGenesis -> Shelley.PParams (MaryEra c) -> PParams (AlonzoEra c)
+  AlonzoGenesis -> Shelley.PParams era -> PParams (AlonzoEra c)
```

Comment:
was this needed?

Reply:
I'd written an instance of "CanStartFromGenesis" that needed it, but should be out from after the rebase, I'll see if this can be backed out'  on the other hand, it's not at all clear to me why it needs to specify `MaryEra` and not be fully general.  PParams is not a type family and has `era` as a phantom parameter.

Reply:
thanks! it's more that I'm not ever expecting to need to translate from any era besides Mary, and I was curious why